media/admin/en-GB.plg_vmshipment_zasilkovna.ini

```diff
@@ -101,6 +107,10 @@ PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_INVALID_TYPE="There is field with invali
 PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_WEIGHT_PRICE_MISSING="Weight price in weight rules is missing"
 PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_WEIGHT_MISSING="Weight in weight rules is missing"
 PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_WEIGHT_EXCEEDED="Weight in weight rules is over specified limit"
+PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_NO_HD_CARRIER_SELECTED="Choose a home delivery carrier"
+PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_HD_CARRIER_IS_OUT_OF_ALLOWED_COUNTRIES="Chosen Home delivery carrier doesn't ship to given country. Choose another carrier."
+PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_HD_CARRIER_NOT_EXISTS="Carrier %s does not exist.."
+PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_HD_CARRIER_REDUNDANT_FOR_PP="No need for Home Delivery carrier, if delivery to Pick up Points is chosen."
```

Comment:
Proč to není v SK překladovém souboru ? Zkopíruj prosím české změny do slovenského souboru.